app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
+      if count_only
+        grouped_rates.length
+      else
+        objects = []
+        grouped_rates.sort.map do |description_group|
+          objects.push(
+            # We identified by the structure Compute_id:Storage_id
+            :id    => description_group[1][0].id.to_s + ":" + description_group[1][1].id.to_s,
```

Comment:
@amaurygonzalez why are we using description_group is an array of chargeback rates, why are we using specific record to build id here, i am not sure i understand this part. It is blowing up for me so maybe i am missing something.  error in UI:
[----] F, [2016-02-04T12:49:48.495832 #3148:3fd98c] FATAL -- : Error caught: [NoMethodError] undefined method `id' for nil:NilClass
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:52:in `block in x_get_tree_roots'
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:47:in `map'
/home/hkataria/dev/manageiq/app/presenters/tree_builder_chargeback_rates.rb:47:in `x_get_tree_roots'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:284:in `x_get_tree_objects'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:250:in `x_build_dynatree'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:190:in `build_tree'
/home/hkataria/dev/manageiq/app/presenters/tree_builder.rb:118:in `initialize'
/home/hkataria/dev/manageiq/app/controllers/chargeback_controller.rb:673:in `new'
/home/hkataria/dev/manageiq/app/controllers/chargeback_controller.rb:673:in `cb_rates_build_tree'
/home/hkataria/dev/manageiq/app/controllers/chargeback_controller.rb:66:in `explorer'